python/composio/utils/warnings.py

```diff
@@ -24,17 +24,14 @@ def _fetch_latest_version():
 
 
 def create_latest_version_warning_hook(version: str):
+    if os.environ.get("COMPOSIO_DISABLE_VERSION_CHECK", "false").lower() != "false":
```

Comment:
Consider extracting this environment variable check into a separate function for better reusability and readability, e.g.:
```python
def is_version_check_disabled():
    return os.environ.get("COMPOSIO_DISABLE_VERSION_CHECK", "false").lower() != "false"
```